keyboards/planck/keymaps/am/config.h

```diff
@@ -0,0 +1,45 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the ifndef, and define with `#pragma once`, and remove the endif at the end of the file?

Reply:
If this is the preferred way to do it, it might be a good idea to change the [default keymap](https://github.com/qmk/qmk_firmware/blob/ade22f8e2c272044ea2f80ff6fe5ca9576858939/keyboards/planck/keymaps/default/config.h#L1) to use that.